spec/package.json

```diff
@@ -10,9 +10,10 @@
     "name": "Viral-Gupta and Neutralinojs community",
     "email": "neutralinojs@gmail.com",
     "url": "https://github.com/neutralinojs-community"
-  },  
+  },
   "license": "MIT",
   "devDependencies": {
+    "@neutralinojs/neu": "^11.2.0",
```

Comment:
I'm not sure this PR is finalized, but adding comments, so they might be helpful. I think we don't need to use the neu CLI program here since we directly test the NodeNeutralino package by running it as a local module. Thanks :tada:

Reply:
Hi @shalithasuranga sir, I am including this package in dev dependencies so we can create a neu app by running `run('neu create test-app')` before running the tests. This ensures the package has an app to run, allowing us to test commands like `app.init()` and similar ones. Do you suggest a better alternative for testing commands like `init` without first creating a neu app?

Reply:
Hi. @viralgupta, Ah okay, I see, we can keep it as it is since you use it only to download binaries :tada: Thanks for your explanation :)